lesson2/README.md

```diff
@@ -54,12 +54,14 @@ Also, please read the following pages that explains the ACID database model and
 
 ## Homework
 
-- Write the necessary SQL statements to complete the functionality of the node task application. This includes, at the minimum, the following functionality:
-    - Adding a new task
-    - Changing the title, due date, or status of a task
-    - Marking a task as completed
-    - Deleting a task
-    - Deleting a user
+Complete the `homework.js` script so that you have a functional node application. In particular, add the necessary code to the functions:
```

Comment:
I think this description is a big improvement. It's much harder to get wrong :smile:

I'm wondering if we should add an extra function for finding a tasks by id. Then we could validate the homework by calling all the functions at the end of the file. Maybe I'm overthinking it...

Reply:
that's not a bad idea at all...